data/init.sql

```diff
@@ -163,14 +153,14 @@ CREATE TABLE IF NOT EXISTS announcements (
 	updated_at TIMESTAMP
 );
 
-CREATE TABLE IF NOT EXISTS course_offerings_instructors(
-	offering_id INT(11) UNSIGNED NOT NULL,
+CREATE TABLE IF NOT EXISTS course_section_instructors(
+	section_id INT(11) UNSIGNED NOT NULL,
 	user_id INT(11) UNSIGNED NOT NULL,
 	FOREIGN KEY (user_id) REFERENCES users(id),
-	FOREIGN KEY(offering_id) REFERENCES course_offerings(id) ON DELETE CASCADE,
+	FOREIGN KEY(section_id) REFERENCES course_sections(id) ON DELETE CASCADE,
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP,
-	PRIMARY KEY (offering_id, user_id)
+	PRIMARY KEY (section_id, user_id)
```

Comment:
@chris7neves @drouba same here, should different sections have different instructors?

Reply:
I agree with you, instructors are assigned to a course, the sections are assigned to TAs.

Reply:
I disagree here, I think different sections can have different instructors.

There are many situations where there will be two or more sections of a course being taught in a semester, and each section will be taught by a different instructor. 

For example, LEC N and LEC M are two lecture sections, and both can have their own professor
However, TUT NI and TUT NJ are tutorial sections, each having a different TA (or can be the same)